src/main/kotlin/resources/task-list.txt

```diff
@@ -0,0 +1 @@
+{"body":"{\"task\":\"a new task\"}","id":1}
```

Comment:
did you want to commit this file?

Reply:
No! Users should not be able to see the stored information; therefore, I'll add it to the `gitignore` file